services/models/istio_validation.go

```diff
@@ -31,10 +31,31 @@ type IstioCheck struct {
 	Path     string `json:"path"`     // String that describes where in the yaml file is the check located
 }
 
+var ObjectTypeMapping = map[string]string{
```

Comment:
Suggestion ObjectTypeSingular ?
(Just in kubernetes/types.go we used a similar map to map istioTypePlurals, I suggest just to follow a similar wording).
No blocker.